snapshot.css

```diff
@@ -5288,377 +5316,6 @@ html:lang(ar)
   width: 100%;
 }
 
-@keyframes draft-packages-likert-scale-legacy-KaizenDraft-LikertScaleLegacy-styles-module__pop--1IPXK {
-  0% {
-    opacity: 0.5;
-    -webkit-transform: scale(1);
-  }
-  100% {
-    opacity: 0;
-    -webkit-transform: scale(1.78);
-  }
-}
-
```

Comment:
All these getting removed makes me suspicious of the snapshot generation...but it's hard to say...

Reply:
Pretty sure the postcss-calc plugin bombed out on this file, so didn't include it.

Reply:
It came back after removing the plugin